packages/launcher/lib/detect.ts

```diff
@@ -88,6 +89,47 @@ function checkBrowser (browser: Browser): Bluebird<(boolean | FoundBrowser)[]> {
   return Bluebird.map([browser], checkOneBrowser)
 }
 
+/**
+ * Retrieves the browser details for the given playwright browser type
+ * @param browserType the playwright browser type
+ * @returns
+ */
+function checkBrowserFamily (browserType: string): Bluebird<(boolean | FoundBrowser)[]> {
+  const playwright = require('playwright')
+
+  return new Bluebird.Promise((resolve, reject) => {
+    //
+    log(`looking up ${browserType} in playwright`)
+    const browserInstance = playwright[browserType]
+
+    browserInstance.launch({}).then((instance: PlaywrightBrowser) => {
```

Comment:
Yeah, I have been asking about this in the Playwright project but it seems to be a bit problematic. I was thinking of running it the first time and then cache the results for runs after that. If the `playwright`-package version changes after that invalidate this cache. That's why I was asking if Cypress code base has some ready code to save such data to the user data directory.

Reply:
There is some existing code to cache things here: https://github.com/cypress-io/cypress/blob/f79bc19814245b0b431cb70a07f8cea6a1e84b42/packages/server/lib/cache.js

Maybe you could use the OS-specific `getVersionString` functions that exist in `launcher` already against `browserInstance.executablePath`, to avoid any extra overhead of `browserInstance.launch`?

Reply:
I had the feeling that some of the Playwright browsers didn't support the `--version` argument for the browser binary. I will try it out again. Maybe it's a macOs specific thing of Playwright there Webkit points to a shell script :)